src/components/security-code-error/security-code-error-controller.ts

```diff
@@ -11,13 +11,25 @@ import {
 } from "../../config";
 
 export function securityCodeInvalidGet(req: Request, res: Response): void {
-  const isNotEmailCode =
-    req.query.actionType !== SecurityCodeErrorType.EmailMaxRetries;
+  let isNotEmailCode = false;
```

Comment:
It's a bit more readable how it was before but perhaps could just change the variable name to `isNotEmailRegsitrationCode`